<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostProcessingClass</class>
 <widget class="QDialog" name="PostProcessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>892</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PostProcessing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_image">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="ImageLabel" name="label_rgb">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string>rgb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_depth">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string>depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cut">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string>cut</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_load">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_play">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cut1">
          <property name="text">
           <string>Cut1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cut2">
          <property name="text">
           <string>Cut2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3d">
     <property name="title">
      <string>3D</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="Viewer" name="graphicsView_kinect">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Viewer2" name="graphicsView_cut">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox_curframe"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_maxframe">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="spinBox_near"/>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_near">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSpinBox" name="spinBox_far"/>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_far">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QGraphicsView</extends>
   <header>Viewer.h</header>
  </customwidget>
  <customwidget>
   <class>Viewer2</class>
   <extends>QGraphicsView</extends>
   <header>Viewer2.h</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>ImageLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PostProcessing.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_curframe</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_curframe</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>PostProcessingClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_near</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_near</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_far</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_far</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_near</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_near</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_far</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_far</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
